src/spender/paths/spender@expenses@check_policies.yaml

```diff
@@ -0,0 +1,48 @@
+post:
+  tags:
+    - Expenses
+  summary: Check expense for policy violation
+  description: |
+    This API checks an expense for policy violations
+  operationId: check_policies_post
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: '../../components/schemas/expense.yaml#/expense_check_policies_in'
+  responses:
```

Comment:
use tick for `UNVIOLATED` and other states - both in this PR as well as the other PR cc @alamirfan

Reply:
`expense_id` to be removed cc @alamirfan

Reply:
see if you can reuse @alamirfan 's full object and ignore just the `expense_id`. If you can't then no other go than copy paste. Just put enough comments at relevant places to guide the future dev on where all he needs to potentially make the change if he is touching one area.

Reply:
`This represent the status of policy computation for an expense against the policy. If the state is UNVIOLATED, it means that the expense is not violating the policy. If the state is VIOLATION_ACTION_SUCCESS, it means that expense is violating the policy and action stated in the object will be applied to expense. However, if the state is VIOLATION_ACTION_FAILURE, it means that the policy is violated for the expense but since some approver was missing or not setup properly, it will fail to take action on the expense` ->

"This represents the status of policy computation for an expense against the policy. If the state is `UNVIOLATED`, it means that the expense is not violating the policy. If the state is `VIOLATION_ACTION_SUCCESS`, it means that the expense is violating the policy and action stated in the object will be applied to the expense when it is saved. However, if the state is `VIOLATION_ACTION_FAILURE`, it means that the policy is violated for the expense but since some approver was missing or not set up properly, it will fail to take action on the expense when it is saved."

Reply:
Department head -> department head

Reply:
policy rule -> expense policy rule

Reply:
see if quotes can be removed from the keyword present inside the tick - both states and status

Reply:
They are enums and the quotes are coming from OpenAPI by default.

Reply:
The double quote is showing in the local preview but when it is published then it not showing.